File: app/src/main/java/com/tcl/john/studymvvm/bean/FoodBean.java
package com.tcl.john.studymvvm.bean;

import android.databinding.ObservableField;

/**
 * Food 实体类
 * Created by ZhangJun on 2017/6/24.
 */

public class FoodBean {

    public final ObservableField<String> description = new ObservableField<>();
    public final ObservableField<String> image = new ObservableField<>();
    public final ObservableField<String> keywords = new ObservableField<>();
    public final ObservableField<String> summary = new ObservableField<>();

}

File: app/src/main/java/com/tcl/john/studymvvm/bean/FoodBean.kt
package com.tcl.john.studymvvm.bean

import android.databinding.ObservableField
import java.io.Serializable

/**
 * Food 实体类
 * Created by ZhangJun on 2017/6/24.
 */


class FoodBean : Serializable {

    companion object {
        private const val serialVersionUID = -8596689964326048249L
    }

    val description = ObservableField<String>()
    val image = ObservableField<String>()

    val keywords = ObservableField<String>()
    val summary = ObservableField<String>()
}
